### PR TITLE
feat(TDP-3537): update subheader styles

### PR DESCRIPTION
**Pull request by charlesnguyen**
**Link to the JIRA issue**
e.g. https://jira.talendforge.org/browse/TDP-3537

**Please check if the PR fulfills these requirements**
- [x] The commit(s) message(s) follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md#commit-message-format)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [ ] The code coverage on new code is > 75 % for backend and > 95% for frontend
- [ ] The new code does not introduce new technical issues (sonar / eslint)
- [x] Functional tests have been performed

**Please check the browsers you've tested on**
- [x] Chrome, Firefox, Safari, Edge, IE11
- [ ] No, that's bad, this PR should not be merged !
- [ ] No, and no need to (backend changes only)

**(Optional) What is the current behavior?**
(Additional information to the Jira)

**(Optional) What is the new behavior?**
(Additional information to the Jira)

**(Optional) Other information**:

### CODE REVIEW

**Comment by build-talend-tdp** [comment]
Pull request test result: [success](https://ci-tdp.datapwn.com/job/tdp_pull-request_master/1060/) [:star:](https://ci-tdp.datapwn.com/job/tdp_pull-request_master/1060/)

File: dataprep-webapp/src/app/components/playground/_playground.scss [hunk 1]
```diff
@@ -59,7 +59,7 @@ $playground-height: '100vh - ' + ($navigation-all-height);
 		}
 
 		#playground-dataset-parameters {
-			@include slidable-top(200px, 0px);
+			@include slidable-top(205px, 0px);
```
Comment (frassinier) [inline dataprep-webapp/src/app/components/playground/_playground.scss]: What's about
```scss
#playground-dataset-parameters {
	@include slidable-top(200px, 0px);

	> .content {
		margin: 10px;
		height: 100%;
	}
}
```

File: dataprep-webapp/src/app/components/playground/_playground.scss [hunk 2]
```diff
@@ -30,7 +30,7 @@ $slide-action-background: $base-background-color;
 
 $navigation-border: 1px;
 $navigation-height: 38px;
-$sub-header-height: 40px;
+$sub-header-height: 38px;
```
Comment (frassinier) [inline dataprep-webapp/src/app/components/playground/_playground.scss]: It looks like to be the same as `$navbar-height` from Bootstrap?

Comment (charlesnguyen) [inline dataprep-webapp/src/app/components/playground/_playground.scss]: yep :)

**Comment by build-talend-tdp** [comment]
Pull request test result: [success](https://ci-tdp.datapwn.com/job/tdp_pull-request_master/1076/) [:star:](https://ci-tdp.datapwn.com/job/tdp_pull-request_master/1076/)

**Comment by build-talend-tdp** [comment]
Pull request test result: [failure](https://ci-tdp.datapwn.com/job/tdp_pull-request_master/1078/) [:x:](https://ci-tdp.datapwn.com/job/tdp_pull-request_master/1078/)

**Pull request merged**